src/assets/translations/fr/main.json

```
@@ -2099,5 +2105,15 @@
       "title": "Définir un nouvel avatar",
       "restoreDefault": "Restaurer à la valeur par défaut"
     }
+  },
+  "foxDiscounts": {
+    "currentFoxPower": "Puissance FOX actuelle",
+    "foxPower": "Puissance FOX",
+    "simulateTitle": "Simulez vos économies FOX",
+    "simulateBody": "Utilisez cette calculatrice pratique pour déterminer le total de vos frais et la remise dont vous bénéficierez sur votre échange. Ajustez simplement les curseurs ci-dessous pour indiquer le montant de vos avoirs FOX et le montant de votre échange en monnaie fiat, et constatez vos économies potentielles !",
```

If we're going for the full translation thing

```suggestion
    "simulateBody": "Utilisez cette calculatrice pratique pour déterminer le total de vos frais et la remise dont vous bénéficierez sur votre échange. Ajustez simplement les curseurs ci-dessous pour indiquer le montant de vos avoirs FOX et le montant de votre échange en monnaie fiduciaire, et constatez vos économies potentielles !",
```

---

Let's go full translation then ;) Thanks for the review